tests/unit/test_column_noise.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+import pandas as pd
+import pytest
+from vivarium.framework.randomness import RandomnessStream
+
+from pseudopeople.noise_functions import (
+    generate_fake_names,
+    generate_nicknames,
+    generate_phonetic_errors,
+    missing_data,
+)
+from pseudopeople.utilities import get_default_configuration
+
+RANDOMNESS = RandomnessStream(
+    key="test_column_noise", clock=lambda: pd.Timestamp("2020-09-01"), seed=0
+)
+
+
+@pytest.fixture(scope="module")
+def string_series():
+    num_simulants = 1_000_000
+    return pd.Series([str(x) for x in range(num_simulants)])
+
+
+@pytest.fixture(scope="module")
+def default_configuration():
+    return get_default_configuration()
+
+
+def test_missing_data(string_series, default_configuration):
+    config = default_configuration["decennial_census"]["zipcode"]["missing_data"]
```

Comment:
Add a todo to change this configuration to be a custom configuration once MIC-3866 is done